packages/@aws-cdk/aws-lambda/lib/lambda-version.ts

```diff
@@ -231,12 +231,12 @@ export class Version extends QualifiedFunctionBase implements IVersion {
   public metric(metricName: string, props: cloudwatch.MetricOptions = {}): cloudwatch.Metric {
     // Metrics on Aliases need the "bare" function name, and the alias' ARN, this differs from the base behavior.
```

Comment:
This comment line is now incorrect with this change. Please make sure it gets updated.